contracts/test/upgrade/MigrationSteps.t.sol

```diff
@@ -0,0 +1,83 @@
+pragma solidity =0.8.18;
+
+import {Test} from "forge-std/Test.sol";
+
+// NOTE TO DEVELOPERS:
+// This checklist test is used to ensure safety of the migration process.
+// These tests serve as a guide to the migration process, step by step.
+// Each step should be marked as complete switching `false` to `true` in `assertTrue`.
+contract MigrationStepsTest is Test {
+    function testStep0_deployContracts() public {
+        // Deploy contracts using CoreDeploy.s.sol and set the contract addresses in:
+        // `UpgradePreparation` and `test_MIGRATION_WARNING_deployedContracts`
+        // Set `triggerMigrationWarning_setDeployedContracts` in `setDeployedContract` to `false` upon completion
+        assertTrue(false);
+    }
+
+    function testStep1_setFinalTokenUnits() public {
+        // By step 1, it is assumed that all the funds have been transferred to the multisig.
+        // Update InitialBountyHelper's token's list to reflect the final bounty units.
+        // Set `triggerMigrationWarning_finalTokens` in `Config` to `false` upon completion
+        assertTrue(false);
+    }
+
+    function testStep2_stopMockingSafeBalances() public {
+        // Remove `mockSafeBalances` from `UpgradePreparation`
+        assertTrue(false);
+    }
+
+    function testStep3_removeForkBlock() public {
+        // Remove `fork` from `UpgradePreparation`.
+        // NOTE: Foundry may cache forked state. Use `forge clean` every time you want to run a clean fork.
+        assertTrue(false);
+    }
+
+    function testStep4_updateCurrentPrice() public {
+        // Update `MIGRATION_WARNING_getCurrentPrice` in `MigrationChecklist` to reflect the latest prices.
+        // Set `triggerMigrationWarning_getCurrentPrice` in `MigrationChecklist` to `false` upon completion
+        assertTrue(false);
+    }
+
+    function testStep5_updateNav() public {
+        // Update `nav` in `MigrationChecklist` to reflect the latest nav.
+        assertTrue(false);
+    }
+
+    function testStep6_removeWarp() public {
+        // Remove `warp` in `UpgradePreparation`
+        assertTrue(false);
+    }
+
```

Comment:
We also have to remove `enableSimulation()` from UpgradePreparation, right?

Reply:
`enableSimulation` can't be removed unless we have all the signatures or have an rpcurl where upgrade is executed. 

`testStep10_tenderlyFork` and `testStep12_mainnet` makes it so that `UpgradedTest` does not call `super.setUp()`, which means it will not call `enableSimulation`.